### PR TITLE
HOLD - DEVELOPER-1757 - Fixed hardcoded links with appropriate base_url links

### PR DESCRIPTION
**Pull request by Dantheman720**
This needs to be tested and checked for broken links.

Replaces https://github.com/redhat-developer/developers.redhat.com/pull/673

### CODE REVIEW

**Comment by Dantheman720** [comment]
retest this please.

File: products/amq/get-started.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@ d. Repeat the previous step to add the following repositories:
 +
 Add two additional repositories. Make certain to click the checkbox to make these repositories active by default.
 +
-* redhat­-developers-­integration­-repository
+* redhat­-site-­integration­-repository
```
Comment (paulrobinson) [inline products/amq/get-started.adoc]: Looks unintended

File: events/javaone/2015/index.html.slim [hunk 2]
```diff
@@ -51,7 +51,7 @@ title: JavaOne
             li
               .session
                 .watch-now
-                  a(href="#{session.url}")
+                  a(href="#{site.base_url}#{session.url}")
```
Comment (paulrobinson) [inline events/javaone/2015/index.html.slim]: Better to include a '/' in between:

      #{site.base_url}/#{session.url}

File: events/javaone/2015/index.html.slim [hunk 3]
```diff
@@ -75,8 +75,8 @@ title: JavaOne
                 .row.talk-info
                   .columns class="#{speaker.url.nil? ? 'large-24' : 'large-13'}"
                     h4 #{speaker.title}
-                  - unless !speaker.url  
-                    .large-11.columns(href="#{speaker.url}")
+                  - unless !speaker.url
+                    .large-11.columns(href="#{site.base_url}#{speaker.url}")
```
Comment (paulrobinson) [inline events/javaone/2015/index.html.slim]: Same here.

**Comment by paulrobinson** [comment]
Otherwise, looks good to merge once these are fixed.

File: products/amq/get-started.adoc (hunk 1 above)
Comment (Dantheman720) [inline products/amq/get-started.adoc]: Strange, I do not remember changing that!

**Pull request merged**